ext/maxminddb.c

```diff
@@ -405,11 +467,29 @@ static zend_class_entry *lookup_class(const char *name TSRMLS_DC)
         zend_error(E_ERROR, "Class %s not found", name);
     }
     return *ce;
+#endif
 }
 
+#ifdef ZEND_ENGINE_3
+static void maxminddb_free_storage(zend_object *object TSRMLS_DC)
+{
+    maxminddb_obj *obj = php_maxminddb_fetch_object(object TSRMLS_CC);
+    if (obj->mmdb != NULL) {
+        MMDB_close(obj->mmdb);
+        efree(obj->mmdb);
+    }
+
+    if (obj->std.properties != NULL) {
+        zend_hash_destroy(obj->std.properties);
+        FREE_HASHTABLE(obj->std.properties);
+    }
+
+    //efree(obj);
```

Comment:
Why is this no longer necessary? If it is really not necessary, could we remove it rather than commenting it out?